timesketch/frontend-ng/src/components/LeftPanel/Stories.vue

```diff
@@ -36,7 +36,7 @@ limitations under the License.
       </v-btn>
 
       <span v-if="!expanded" class="float-right" style="margin-right: 10px">
-        <small v-if="meta.stories.length"
+        <small v-if="meta.stories"
           ><strong>{{ meta.stories.length }}</strong></small
```

Comment:
This change leads to some odd behaviour where the counter will appear and disappear when clicked and no story is there. See gif below:
![story-test](https://github.com/google/timesketch/assets/99879757/29b59d65-2387-4e1f-8f29-89431e840d6f)

Reply:
fixed